server/src/main/java/org/apache/druid/segment/loading/StorageLocation.java

```diff
@@ -134,6 +134,8 @@ public synchronized File reserve(String segmentFilePathToAdd, String segmentId,
       currSizeBytes += segmentSize;
       return segmentFileToAdd;
     } else {
+      log.error("No space is reserced or druid.segmentCache.locations is reached maxSize in location [%s], " +
```

Comment:
A warning message is logged in `canHandle` that describes the specific issue it ran into in case `canHandle` returns false. Is there something in that warning message that could be improved?